optimism2/aave/tokens.sql

```diff
@@ -0,0 +1,67 @@
+CREATE TABLE IF NOT EXISTS aave.tokens (   
+    symbol text,
```

Comment:
just for consistency, add version here as well for optimism (similar to that for polygon)